draft-ietf-cbor-update-8610-grammar.md

```diff
@@ -299,9 +300,61 @@ It does require some care when copy-pasting into CDDL models from ABNF
 that contains single quotes (which may also hide as apostrophes
 in comments); these need to be escaped or possibly replaced by `%x27`.
 
-Finally, our approach would lend support to extending `bsqual` in CDDL
+Finally, our approach lends support to extending `bsqual` in CDDL
 similar to the way this is done for CBOR diagnostic notation in {{-edn}}.
+(Note that the processing of string literals now is quite similar between
+CDDL and EDN, except that CDDL has "`;`"-based end-of-line comments, while EDN has
+two comment syntaxes, in-line "`/`"-based and end-of-line "`#`"-based.)
+
+The CDDL example in {{string-examples}} demonstrates various escaping
+techniques.
+Obviously in the literals for `a` and `x`, there is no need to escape
+the second character, an `o`, as `\u{6f}`; this is just for demonstration.
+Similarly, as shown in `c` and `z` there also is no need to escape the
+`🁁` or `⌘`, but escaping them may be convenient in order to limit the character
+repertoire of a CDDL file itself to ASCII {{-ascii}}.
 
+~~~ cddl
+start = [a, b, c, x, y, z]
+
+; "🁁", DOMINO TILE HORIZONTAL-02-02, and
+; "⌘", PLACE OF INTEREST SIGN, in a text string:
+a = "D\u{6f}mino's \u{1F041} + \u{2318}"  ; \u{}-escape 3 chars
+b = "Domino's \uD83C\uDC41 + \u2318"      ; escape JSON-like
+c = "Domino's 🁁 + ⌘"                      ; unescaped
```

Comment:
At least GitHub's code viewer renders 🁁 wider than a single character width, offsetting the comments. If that happens with the IETF rendered view as well, 🂁 might avoid the issue.

Reply:
🁁 is the result of a long journey to get a non-BMP character into RFC 9535.
🁳 probably would be accepted by the RFC editor as well, so I'll make this small esthetic change.